src/sentry/data_export/processors/issues_by_tag.py

```diff
@@ -94,7 +94,7 @@ def serialize_row(item, key):
             result["ip_address"] = euser.ip_address if euser else ""
         return result
 
-    def get_raw_data(self, offset=0):
+    def get_raw_data(self, limit=1000, offset=0):
```

Comment:
Does this need a default value if it isn't called directly by the data export tasks?

Reply:
It may make more sense to take out the default argument and require the caller to pass it in as this isn't used anywhere else currently. However, I put it as a default argument to be consistent with the other methods that already exists like `src/sentry/tagstore/snuba/backend.py`. I can take out the default argument if you'd like.

Reply:
It isn't a big concern, having multiple default arguments means you need to remember to keep them synced up, and can make it harder to track down errors when you fail to pass a parameter from the 'public' method.